src/incubator/Dialog/DialogHeader.tsx

```diff
@@ -4,11 +4,12 @@ import {StyleSheet} from 'react-native';
 import {asBaseComponent} from '../../commons/new';
 import {Spacings, Colors, BorderRadiuses, Dividers} from 'style';
 import View from '../../components/view';
-import HeaderContent from './HeaderContent';
+import Text from '../../components/text';
 import {DialogHeaderProps} from './types';
 
 const DialogHeader = (props: DialogHeaderProps = {}) => {
   const {text = {}, renderContent, showKnob = true, showDivider = true, ...others} = props;
+  const {title, titleStyle, titleProps, subtitle, subtitleStyle, subtitleProps} = text;
```

Comment:
Does dialog header has a `text` prop that contain all of these? 
I think `text` is not a good name here... 

Is it internal or an API we're obligated to?

Reply:
No, this is new, I agree it's not a very good name :/
I wanted to separate some props, do you prefer another name or to put them all under `headerProps?: DialogHeaderProps` (which already exists and has a few props)?

Reply:
Ping

Reply:
Done